src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+const TodoForm = props => {
+  return (
+    <form>
+      <input
+        onChange={props.onChangeTodo}
+        type="text"
+        name="task"
+        value={props.value}
+        placeholder="add a task"
+      />
```

Comment:
You can add `required` here so that your app will prompt users if they try to add an empty todo.